filesystems/filesystems.tex

```diff
@@ -517,23 +533,25 @@ \subsection{Metadata}
 int lstat(const char *path, struct stat *buf);
 \end{lstlisting}
 
-For example, you can use \keyword{fstat} to find out the meta-information about a file if you already have an file descriptor associated with that file.
+For example, you can use \keyword{fstat} to learn about file metadata if you already have a file descriptor associated with that file.
 
 \begin{lstlisting}[language=C]
-FILE *file = fopen("notes.txt", "r");
-int fd = fileno(file); /* Just for fun - extract the file descriptor from a C FILE struct */
-struct stat s;
-fstat(fd, & s);
-printf("Last accessed %s", ctime(&s.st_atime));
+    FILE *file = fopen("notes.txt", "r");
+    int fd = fileno(file); /* Just for fun - extract the file descriptor from a C FILE struct */
+    struct stat s;
+    fstat(fd, & s);
+    printf("Last accessed %s", ctime(&s.st_atime));
```

Comment:
When we put in spacing to a lstlisting environment, it creates an odd indent in the markdown and the pdf.